no-computer.md

```diff
@@ -58,12 +116,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Braille that represents the English language has already been created!
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+ I’m not sure what this question means, since Braille already exists. But if one were to represent another human language with Braille, that might be a challenging undertaking since it seems like many of the combinations of raised and lowered dots are used. It also wouldn’t be intuitive to design an entirely new system of Braille that doesn’t expand upon any of the symbols and patterns of existing Braille. It would be great if this new Braille sub-language would be syntactically similar enough to existing Braille sub-languages such that it would be intuitive for those who are learning the new sub-language, but the new sub-language must also be careful to not be too similar to existing sub-languages to the point where people could find the similarities indistinguishable and easy to mix up. 
+
```

Comment:
I feel like if you were representing an entirely different human language in Braille, you wouldn't need to be concerned about users mixing it up with, for example, English Braille (much like how we can immediately identify writing from another language visually).  Were you referring to different types of "sub-languages?"